src/frontend/github-oauth.ts

```diff
@@ -1,187 +0,0 @@
-import crypto from "crypto";
```

Comment:
not sure why this was considered deleted as it was moved to /src/routes/github, but changed a bit to be it's own router